mlflow/data/tensorflow_dataset.py

```diff
@@ -23,18 +23,21 @@ def __init__(
         self,
         data,
         source: DatasetSource,
+        targets=None,
         name: Optional[str] = None,
         digest: Optional[str] = None,
     ):
         """
-        :param data: A Tensorflow dataset or Tensorflow tensor.
+        :param data: A Tensorflow dataset or tensor.
         :param source: The source of the Tensorflow dataset.
+        :param targets: A Tensorflow dataset or tensor containing dataset targets. Optional
```

Comment:
```suggestion
        :param data: A TensorFlow dataset or tensor.
        :param source: The source of the Tensorflow dataset.
        :param targets: A TensorFlow dataset or tensor containing dataset targets. Optional.
```

Can you check the capitalization of TensorFlow throughout your docstrings and adjust if needed?